rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples$ git clone https://github.com/rodrigo-pacheco/RoboticsAcademy.git
Clonar en «RoboticsAcademy»...
remote: Enumerating objects: 4953, done.
remote: Total 4953 (delta 0), reused 0 (delta 0), pack-reused 4953
Receiving objects: 100% (4953/4953), 6.93 MiB | 2.33 MiB/s, done.
Resolving deltas: 100% (3165/3165), done.
Comprobando la conectividad… hecho.
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples$ ls
RoboticsAcademy
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples$ cd RoboticsAcademy/
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy$ ls
docs  exercises  README.md
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy$ cd exercises/
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises$ ls
2D visual odometry with RGBD  bump_and_go   color_filter     follow_face  follow_turtlebot   laser_loc           rescue_people   visual_lander
3d_reconstruction             car_junction  drone_cat_mouse  follow_line  global_navigation  obstacle_avoidance  vacuum_cleaner
autopark                      chrono        drone_hangar     follow_road  labyrinth_escape   position_control    vacuum_slam
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises$ cd follow_line/
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises/follow_line$ ls
follow_line_conf.yml  follow_line.py  generateGUI  gui  interfaces  MyAlgorithm.py  README.md  resources  resources_rc.py
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises/follow_line$ roslaunch /opt/jderobot/share/jderobot/launch/f1.launch
Traceback (most recent call last):
  File "/opt/ros/kinetic/bin/roslaunch", line 34, in <module>
    import roslaunch
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/__init__.py", line 48, in <module>
    import rospkg
ImportError: No module named 'rospkg'
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises/follow_line$ ^C
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises/follow_line$ sudo apt-get install python-rospkg
[sudo] password for rodrigo: 
Lo sentimos, vuelva a intentarlo.
[sudo] password for rodrigo: 
Leyendo lista de paquetes... Hecho
Creando árbol de dependencias       
Leyendo la información de estado... Hecho
python-rospkg ya está en su versión más reciente (1.1.7-100).
fijado python-rospkg como instalado manualmente.
Los paquetes indicados a continuación se instalaron de forma automática y ya no son necesarios.
  libllvm4.0 libllvm4.0:i386 libllvm5.0:i386 libqmi-glib1 libqpdf17
Utilice «sudo apt autoremove» para eliminarlos.
0 actualizados, 0 nuevos se instalarán, 0 para eliminar y 376 no actualizados.
13 no instalados del todo o eliminados.
Se utilizarán 0 B de espacio de disco adicional después de esta operación.
¿Desea continuar? [S/n] s
Configurando jderobot-zeroc-ice-python (5.6.6) ...
Traceback (most recent call last):
  File "/usr/bin/pip2", line 9, in <module>
    load_entry_point('pip==8.1.1', 'console_scripts', 'pip2')()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2750, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('console_scripts', 'pip2') not found
Traceback (most recent call last):
  File "/usr/bin/pip2", line 9, in <module>
    load_entry_point('pip==8.1.1', 'console_scripts', 'pip2')()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2750, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('console_scripts', 'pip2') not found
Traceback (most recent call last):
  File "/usr/bin/pip2", line 9, in <module>
    load_entry_point('pip==8.1.1', 'console_scripts', 'pip2')()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2750, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('console_scripts', 'pip2') not found
dpkg: error al procesar el paquete jderobot-zeroc-ice-python (--configure):
 el subproceso instalado el script post-installation devolvió el código de salida de error 1
dpkg: problemas de dependencias impiden la configuración de jderobot-comm-python:
 jderobot-comm-python depende de jderobot-zeroc-ice-python; sin embargo:
 El paquete `jderobot-zeroc-ice-python' no está configurado todavía.
  El paquete `jderobot-zeroc-ice-python' que provee `jderobot-zeroc-ice-python' aún no está configurado.

dpkg: error al procesar el paquete jderobot-comm-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-easyice-python:
 jderobot-easyice-python depende de jderobot-zeroc-ice-python; sin embargo:
 El paquete `jderobot-zeroc-ice-python' no está configurado todavía.
  El paquete `jderobot-zeroc-ice-python' que provee `jderobot-zeroc-ice-python' aún no está configurado.

dpkg: error al procesar el paquete jderobot-easyice-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-colortuner-pythNo se escribió un informe «apport» porque el mensaje de error indica que es un mensaje de error asociado a un fallo previo.
                                                         No se escribió un informe «apport» porque el mensaje de error indica que es un mensaje de error asociado a un fallo previo.
                              No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                                                       No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                          No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                                                                                   No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                      on:
 jderobot-colortuner-python depende de jderobot-comm-python; sin embargo:
 El paquete `jderobot-comm-python' no está configurado todavía.
 jderobot-colortuner-python depende de jderobot-easyice-python; sin embargo:
 El paquete `jderobot-easyice-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-colortuner-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-pantiltteleop-python:
 jderobot-pantiltteleop-python depende de jderobot-comm-python; sin embargo:
 El paquete `jderobot-comm-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-pantiltteleop-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-parallelice-python:
 jderobot-parallelice-python depende de jderobot-zeroc-ice-python; sin embargo:
 El paquete `jderobot-zeroc-ice-python' no está cNo se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                                                                          No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                             onfigurado todavía.
  El paquete `jderobot-zeroc-ice-python' que provee `jderobot-zeroc-ice-python' aún no está configurado.

dpkg: error al procesar el paquete jderobot-parallelice-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-uav-viewer-python:
 jderobot-uav-viewer-python depende de jderobot-parallelice-python; sin embargo:
 El paquete `jderobot-parallelice-python' no está configurado todavía.
 jderobot-uav-viewer-python depende de jderobot-easyice-python; sin embargo:
 El paquete `jderobot-easyice-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-uav-viewer-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-visualstates-python:
 jderobot-visualstates-python depende de jderobot-comm-python; sin embargo:
 El paquete `jderobot-comm-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-visualstates-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-tools:
 jderobot-tools depende de jderobot-colortuner-python; sin embargo:
 El paquete `jderobot-colortuner-python' no está configurado todavía.
 jderobot-tools depende de jderobot-pantiltteleop-python; sin embargo:
 El paquete `jderobot-pantiltteleop-python' no está configurado todavía.
 jderobot-tools depende de jderobot-uav-viewer-python; sin embargo:
 El paquete `jderobot-uav-viewer-python' no está configurado todavía.
 jderobot-tools depende de jderobot-visualstates-python; sin embargo:
 El paquete `jderobot-visualstates-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-tools (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-libs:
 jderobot-libs depende de jderobot-easyice-python; sin embargo:
 El paquete `jderobot-easyice-python' no está configurado todavía.
 jderobot-libs depende de jderobot-comm-python; sin embargo:
 El paquete `jderobot-comm-python' no está configurado todavía.
 jderobot-libs depende de jderobot-parallelice-python; sin embargo:
 El paquete `jderobot-parallelice-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-libs (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-basic-component-python:
 jderobot-basic-component-python depende de jderobot-comm-python; sin embargo:
 El paquete `jderobot-comm-python' no está configurado todavía.

No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                         No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                            dpkg: error al procesar el paquete jderobot-basic-component-python (--configure):
 problemas de dependencias - se deja sin configurar
dpkg: problemas de dependencias impiden la configuración de jderobot-examples:
 jderobot-examples depende de jderobot-basic-component-python; sin embargo:
 El paquete `jderobot-basic-component-python' no está configurado todavía.

dpkg: error al procesar el paquete jderobot-examples (--configure):
 problemas de dependencias - se deja sin configurar
No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                         dpkg: problemas de dependencias impiden la configuración de jderobot:
 jderobot depende de jderobot-tools; sin embargo:
 El paquete `jderobot-tools' no está configurado todavía.
  El paquete `jderobot-tools' que provee `jderobot-tools' aún no está configurado.
 jderobot depende de jderobot-libs; sin embargo:
 El paquete `jderobot-libs' no está configurado todavía.
  El paquete `jderobot-libs' que provee `jderobot-libs' aún no está configurado.
 jderobot depende de jderobot-examples; sin embargo:
 El paquete `jderobot-examples' no está configurado todavía.
  El paquete `jderobot-examples' que provee `jderobot-examples' aún no está configurado.

dpkg: error al procesar el paquete jderobot (--configure):
 problemas de dependencias - se deja sin configurar
No se escribió ningún informe «apport» porque ya se ha alcanzado el valor de «MaxReports»
                                                                                         Se encontraron errores al procesar:
 jderobot-zeroc-ice-python
 jderobot-comm-python
 jderobot-easyice-python
 jderobot-colortuner-python
 jderobot-pantiltteleop-python
 jderobot-parallelice-python
 jderobot-uav-viewer-python
 jderobot-visualstates-python
 jderobot-tools
 jderobot-libs
 jderobot-basic-component-python
 jderobot-examples
 jderobot
E: Sub-process /usr/bin/dpkg returned an error code (1)
rodrigo:~/Documentos/DG_ITT_IAA/TFG DRONES/GITHUB/Examples/RoboticsAcademy/exercises/follow_line$ 

